wremnants/histselections.py

```diff
@@ -15,39 +15,35 @@
     "ptll_mll" : "nominal",
 }
 
-def fakeHistABCD(h, low_PU=False,
-                 boolMT=True, thresholdMT=40.0, axisNameMT="mt", integrateMT=True):
+def fakeHistABCD(h, thresholdMT=40.0, fakerate_integration_axes=["ptll"], axis_name_mt="mt", integrateMT=False):
```

Comment:
Shouldn't we integrate over all non-pt/eta/mt/iso axes by default?

Reply:
I noticed that it didn't work. If we integrate over all non-pt/eta/mt/iso/charge axes by default we end up integrating over the systematics axes as well, which we don't want. I don't see any other reason than to specify directly which axes should be integrated over ...

Reply:
Hmmm, good point

Reply:
I guess it should probably be possible to propagate this through the CardTool and the loadHistsForDatagroups function, right? From the cardTool should probably integrate over the axes in the fitAxes excluding pt/eta/charge

Reply:
Okay I think this works, it's implemented